lib/crawlers/base_crawler.rb

```diff
@@ -0,0 +1,153 @@
+require_relative "../priority_spidr/priority_spidr"
```

Comment:
You shouldn't need to require_relative here I don't believe

Reply:
The tests fail when if I use `require 'priority_spidr'` instead.